### ISSUE TITLE
a11y: Color Contrast - Change Password

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13706]
Insufficient color contrast for the borders of checkboxes

Insufficient color contrast for standard text

## Actual result

The mentioned checkbox border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

## Instances

Change Password
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
- [ ] Delete all existing API keys and Generate a random password 

Cluster Appearance
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
- [ ] Use custom badge, Show cluster comment, Badge background color 

Pod Security Admission: Create
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
- [ ] Usernames, RuntimeClasses and Namespaces 

Git Repo - Add Repository
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
- [ ] Enable Self-Healing and Always Keep Resources 

Cluster Drivers
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Alibaba ACK
- [ ] Amazon EKS
- [ ] Azure AKS
- [ ] Baidu CCE and so on unselected checkboxes

Foreground color: #DCDEE7
Background color: #F4F5FA
Contrast ratio: 1.23:1 
Main Content 
- [ ] Alibaba ACK
- [ ] Amazon EKS
- [ ] Azure AKS
- [ ] Baidu CCE and so on unselected checkboxes (on hover) 

Charts
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Show deprecated apps checkbox

Users Retention Settings 
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Disable user accounts after an inactivity period (duration since last login)  and Delete user accounts after an inactivity period (duration since last login)  checkboxes. 

User: Create
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
 - [ ] Ask user to change their password on next login, Generate a random password, Administrator, Restricted Administrator, Standard User and so on checkboxes. 

Add Extensions repositories 
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Add Partners Extensions Repository checkbox. 

Banners
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Show Banner in Header, Show Banner in Footer, Show Consent Banner on Login Screen, Show Login Consent as a modal dialog, Show custom login error, Bold, Italic and Underline checkboxes 

Branding
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use a Custom Favicon, Use a Custom Color and Use a Custom Link Color checkboxes.

Home Links
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Unselected Show checkbox

Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio: 3.16:1
- [ ] Current Password (in typing state)
- [ ] New Password (in typing state)
- [ ] Confirm Password (in typing state)
- [ ] Show 
- [ ] Hide (available after activating show) 
- [ ] Cancel
- [ ] Suggest a password (available after checking Generate a random password checkbox) 

Foreground color: #3D98D3
Background color: #DCDEE7
Contrast ratio: 2.35:1 
- [ ] Copy (available after checking Generate a random password checkbox) 

Foreground color: #F64747
Background color: #FEE4E4
Contrast ratio: 2.94:1
- [ ] Invalid current password, Password must be at least 12 characters

Foreground color: #646D93
Background color: #DCDEE7 
Contrast ratio: 3.76:1
- [ ] Generated Password (available after checking the Generate a random password checkbox)

## Expected result

The contrast ratio between the border of checkbox control and its adjacent color should be sufficient.

**Comment by rak-phillip** [comment]
Fails the VPAT. No notes. Original comment:

> The mentioned checkbox border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 102
